submissions/xT0PERx/Friends app/js/index.js

```diff
@@ -0,0 +1,177 @@
+const allFriends = {
+  currentList: [],
+  changeList: [],
+  numberOfFriends: 72,
+};
+
+const REQUEST_URL = "https://randomuser.me/api/1.3/";
+const ALL_USERS_CARDS_FILTERS = allFriends.changeList;
+const ALL_USERS_CARDS = allFriends.currentList;
+const USERS_LIST = document.getElementById("usersList");
+const FILTERS_MENU = document.getElementById("filters_menu");
+const ABC_BUTTON_FILTER = document.getElementById("Abc");
+const ZYX_BUTTON_FILTER = document.getElementById("Zyx");
+const MALE_BUTTON_FILTER = document.getElementById("male");
+const FEMALE_BUTTON_FILTER = document.getElementById("female");
+const YOUNGER_FILTER_BUTTON = document.getElementById("younger-ftr");
+const SENIOR_FILTER_BUTTON = document.getElementById("senior-ftr");
+const RESET_BUTTON = document.getElementById("reset");
+const SEARCH_INPUT = document.querySelector(".searcher");
+
+function initApp() {
+  getUsers();
+}
+
+initApp();
+
+function getUsers() {
+  const url = `${REQUEST_URL}?results=${allFriends.numberOfFriends}`;
+  fetch(url)
+    .then((res) => {
+      if (res.ok) {
+        return res.json();
+      } else {
+        throw new Error("Что-то не так с сервером...");
+      }
+    })
+    .then((data) => {
+      saveUsers(data.results);
+      createUsersCards(ALL_USERS_CARDS);
+    })
+    .catch((err) => {
+      console.error(err.message);
+    });
+}
+
+function saveUsers(users) {
+  users.forEach(function (el) {
+    ALL_USERS_CARDS.push(el);
+    ALL_USERS_CARDS_FILTERS.push(el);
+  });
+}
+
+function createUsersCards(users) {
+  const lists = document.createDocumentFragment();
+
+  users.forEach(function (el) {
+    const card = document.createElement("li");
+    card.classList.add("userCard");
+    card.innerHTML = `<span class="name">${el.name.first} ${el.name.last}</span>
+    <img src="${el.picture.large}">
+    <span class="age">Age: ${el.dob.age}</span>
+    <span class="location">Location: ${el.location.city}</span>`;
```

Comment:
Do I understand this point of the comment correctly? 
___Prefer template strings to “createElement” API___
This means that it is better for me to create elements not through a piece of  HTML code, but using a command such as __document.createElement()__? and then __innerHTML="bla-bla"__?

Reply:
No, vice-versa. Template strings should be used